cmake-template.SlackBuild

```diff
@@ -115,8 +115,10 @@ cd build
     -DCMAKE_C_FLAGS:STRING="$SLKCFLAGS" \
     -DCMAKE_CXX_FLAGS:STRING="$SLKCFLAGS" \
     -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_DOCDIR="doc/$PRGNAM-$VERSION" \
+    -DCMAKE_INSTALL_INFODIR=info \
+    -DCMAKE_INSTALL_MANDIR=man \
```

Comment:
If a relative path is given for any of these variables, the computed installation path is relative to `CMAKE_INSTALL_PREFIX`, as documented in `/usr/share/cmake-*/Modules/GNUInstallDirs.cmake`.